pkg/private/deb/deb.bzl

```diff
@@ -107,6 +108,12 @@ def _pkg_deb_impl(ctx):
     else:
         fail("Neither description_file nor description attribute was specified")
 
+    # Files for stamping variables
+    if ctx.attr.private_stamp_detect:
```

Comment:
This should follow the logic of what the others do. Where there is a stamp parameter which can be 1, 0, or -1, and we only use the stamp dectect if -1.